doc/qbk/05_performance.qbk

```diff
@@ -0,0 +1,282 @@
+each configuration. All but the median two samples are discarded, with
+the remaining samples averaged to produce a single number which is reported
+as the benchmark result.
+
+The input files, available in the bench/data directory, are laid out thusly:
+
+[table Input Files
+[
+    [Name]
+    [Size]
+    [Description]
+][
+    [[link json.performance.parse_apache_builds_json [*apache_builds.json]]]
+    [125KB]
+    [
+        Data from the Apache Jenkins installation.
+    ]
+][
+    [[link json.performance.parse_canada_json [*canada.json]]]
+    [2.2MB]
+    [
+        The largest file, containing a large number of 2-element
+        arrays holding floating-point coordinate pairs.
+    ]
+][
+    [[link json.performance.parse_citm_catalog_json [*citm_catalog.json]]]
+    [1.69MB]
+    [
+        A large JSON with a variety of nesting, types, and lengths.
+    ]
+][
+    [[link json.performance.parse_github_events_json [*github_events.json]]]
+    [64KB]
+    [
+        An export of data from the Github Events API.
+    ]
+][
+    [[link json.performance.parse_gsoc_2018_json [*gsoc-2018.json]]]
+    [3.25MB]
+    [
+        Google Summer of Code 2018 data.
+    ]
+][
+    [[link json.performance.parse_instruments_json [*instruments.json]]]
+    [216KB]
+    [
+        An array of large objects.
+    ]
+][
+    [[link json.performance.parse_marine_ik_json [*marine_ik.json]]]
+    [2.91MB]
+    [
+        A three.js example model serialized to JSON.
+    ]
+][
+    [[link json.performance.parse_mesh_json [*mesh.json]]]
+    [707KB]
+    [
+        A JSON representing a 3D mesh. Contains many floating-point numbers.
+    ]
+][
+    [[link json.performance.parse_mesh_pretty_json [*mesh.pretty.json]]]
+    [1.54MB]
+    [
+        mesh.json with whitespace added.
+    ]
+][
+    [[link json.performance.parse_numbers_json [*numbers.json]]]
+    [147KB]
+    [
+        A array containing only floating-point numbers.
+    ]
+][
+    [[link json.performance.parse_random_json [*random.json]]]
+    [499KB]
+    [
+        A JSON with lots of Cyrillic characters.
+    ]
+][
+    [[link json.performance.parse_twitter_json [*twitter.json]]]
+    [617KB]
+    [
+        An export of data from Twitter's API.
+    ]
+][
+    [[link json.performance.parse_twitterescaped_json [*twitterescaped.json]]]
+    [550KB]
+    [
+        twitter.json with whitespace removed and non-ASCII
+        characters replaced with Unicode escapes.
+    ]
+][
+    [[link json.performance.parse_update_center_json [*update-center.json]]]
+    [521KB]
+    [
+        An export of data from Twitter's API.
+    ]
+]
+]
+
+Hardware used for testing: [*Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz],
+Windows 10, 32GB RAM. 
+
+Compilers and optimization flags: gcc 8.1 (-O3), clang 10.0 (-O3), and msvc 19.26 (/O2).
+
+Thanks to SSE2 optimized string processing algorithms from Peter Dimov,
+Boost.JSON significantly outperforms all other libraries for parsing
+and serializing unescaped strings:
```

Comment:
non-sterile (yes I wrote it). we can mention this but it should be sterile, e.g.

> Boost.JSON uses SSE2 to accelerate processing of unescaped strings

Reply:
We use SSE2 for much more than just that

Reply:
Yes I know, I'm not saying to use my text verbatim. You can just remove my text (Lines 189-193), and add something later if you think of something that is relevant and sterile.

Reply:
I edited it myself to remove my non-sterile language